.github/workflows/publish-client.yml

```
@@ -0,0 +1,37 @@
+  pull_request:
+
+jobs:
+  publish-client-prerelease:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup environment
+        uses: ./.github/actions/setup-env
+
+      - name: Setup node
+        uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)